openassessment/locale/config.yaml

```diff
@@ -74,11 +74,19 @@ dummy_locales:
 
 # Directories we don't search for strings.
 ignore_dirs:
```

Comment:
See https://github.com/edx/i18n-tools/blob/master/i18n/extract.py#L100 for how `i18n_tool` explicitly uses the `ignore_dirs`